assets/requirements/windows_cpu.compiled

```diff
@@ -140,7 +140,7 @@ networkx==3.4.2
     #   --override assets/override.txt
     #   torch
     # from https://pypi.org/simple
-numpy==2.1.3
+numpy==1.26.4
```

Comment:
Why is numpy downgraded in this PR? Please link to a main repo issue if it's doing a sync of deps from there.